cluster-autoscaler/processors/provreq/injector_test.go

```diff
@@ -66,9 +66,12 @@ func TestProvisioningRequestPodsInjector(t *testing.T) {
 		LastTransitionTime: metav1.NewTime(minAgo),
 	}
 
+	// provisioningClassPrefix := "test-"
+
 	podsA := 10
 	newProvReqA := testProvisioningRequestWithCondition("new", podsA, v1.ProvisioningClassCheckCapacity)
 	newAcceptedProvReqA := testProvisioningRequestWithCondition("new-accepted", podsA, v1.ProvisioningClassCheckCapacity, accepted)
+	newProvReqAPrefixed := testProvisioningRequestWithCondition("new-prefixed", podsA, "test-check-capacity.autoscaling.x-k8s.io")
```

Comment:
perhaps have "prefix" in the name of the prefix, to make the intent more clear?

Could be:
`test-prefix.check-capacity.autoscaling.x-k8s.io`